dashboard/app/models/user_permission.rb

```diff
@@ -22,9 +22,6 @@ class UserPermission < ActiveRecord::Base
     # Grants access to creating professional development workshops.
     CREATE_PROFESSIONAL_DEVELOPMENT_WORKSHOP =
```

Comment:
Note: this one is also deprecated, and actually not [referenced anywhere](https://github.com/code-dot-org/code-dot-org/search?q=CREATE_PROFESSIONAL_DEVELOPMENT_WORKSHOP&unscoped_q=CREATE_PROFESSIONAL_DEVELOPMENT_WORKSHOP) (except an old oneoff script). It was used in the old K5 (CSF) workshop dashboard which we removed earlier this year.

Reply:
Does somebody else want to take this one?

![image](https://user-images.githubusercontent.com/1615761/41929354-9ba35220-792c-11e8-8166-51f4b970855f.png)

Reply:
I'll take this one